src/modules/lorem/index.ts

```diff
@@ -326,10 +332,35 @@ export class LoremModule {
    * // Quis ut dolor dolores facilis possimus tempore voluptates.
    * // Iure nam officia optio cumque.
    * // Dolor tempora iusto.'
+   * faker.lorem.text(14) // 'Doloribus aut.' (14 characters)
+   * faker.lorem.text({length: 14}) // 'Doloribus aut.' (14 characters)
+   * faker.lorem.text({length: {min: 10, max: 15}}) // 'autem non quis.' (15 characters)
+   * faker.lorem.text({length: {min: 10, max: 12}}) // 'autem non.' (10 characters)
+   * faker.lorem.text({length: {min: 3, max: 10}}) // 'autem.' (6 characters)
```

Comment:
I dont think we need the explicit character information at the end. We also dont need that many repetitive examples:

```suggestion
   * faker.lorem.text(14) // 'Doloribus aut.'
   * faker.lorem.text({length: 14}) // 'Doloribus aut.'
   * faker.lorem.text({length: {min: 10, max: 15}}) // 'autem non quis.'
```